File: Modul-6/Soal-3/PRAK603-2410817320001-NazlaSalsabila.py
n1, n2 = map(int, input("Masukkan n1 dan n2: ").split())
if n1 != n2:
    print("Jumlah tidak sama")
else:
    print(f"Masukkan {n1} angka untuk baris pertama:", end=" ")
    baris1 = list(map(int, input().split()))
    print(f"Masukkan {n2} angka untuk baris kedua:", end=" ")
    baris2 = list(map(int, input().split()))
    hasil = [baris1[i] * baris2[i] for i in range(n1)]
    print("Hasil perkalian:")
    print(" ".join(map(str, hasil)))

File: Modul-6/Soal-3/PRAK603-2410817320001-NazlaSalsabila.c
#include <stdio.h>

int main() {
    int n1, n2;
    printf("Masukkan n1 dan n2: ");
    scanf("%d %d", &n1, &n2);
    
    if (n1 != n2) {
        printf("Jumlah tidak sama\n");
    } else {
        int baris1[n1], baris2[n2];
        printf("Masukkan %d angka untuk baris pertama: ", n1);
        for (int i = 0; i < n1; i++) {
            scanf("%d", &baris1[i]);
        }
        
        printf("Masukkan %d angka untuk baris kedua: ", n2);
        for (int i = 0; i < n2; i++) {
            scanf("%d", &baris2[i]);
        }
        
        printf("Hasil perkalian:\n");
        for (int i = 0; i < n1; i++) {
            printf("%d ", baris1[i] * baris2[i]);
        }
    }
    
    return 0;
}